etc/kayobe/kolla/config/magnum/policy.yaml

```diff
@@ -0,0 +1,4 @@
+# Retrieve CA information about the given bay/cluster.
```

Comment:
Only policy.json is supported by kolla-ansible in pike. Yaml should work in queens.